<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbl_score">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next â€“&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_question">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>411</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Question</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_eingabe">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_score_value">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ultimate Quiz</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_correctAnswers">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Antworten korrigieren</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>txt_eingabe</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_next</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
